doc/source/examining/loading_data.rst

```diff
@@ -63,6 +63,19 @@ The user has two ways to control displayed units, through
   units_override = dict(length_unit=(100., 'au'), mass_unit=yt.units.mass_sun)
   ds = yt.load("output0010.dat", units_override=units_override, unit_system="mks")
 
+  To ensure consistency with normalisations as used in AMRVAC we only allow
+  overriding a maximum of three units. Allowed unit combinations at the moment are
```

Comment:
This is at the same indent level as the code above and will be interpreted as a continuation of the above code block. You can see for yourself in the rich view of this file on github.

Reply:
Indeed... Fixed, amended the commit